meetings/Agenda_2021-03-24.md

```diff
@@ -0,0 +1,12 @@
+## Meeting Agenda, Mar 24 2021
+
+1. **Debrief** (20 min)
+ * What did we feel went well about this past semester in terms of lessons?
+ * What could have been better about the lessons this past semester? 
+ * How did we feel about the change in schedule this semester?
+ * How did we feel about co-working? 
+
+2. **Summer/future plans** (20-25 min) 
```

Comment:
Carpentries workshop in the summer, May?

Reply:
GAH I knew I was forgetting something thank you